Packs/MicrosoftExchangeOnline/Integrations/SecurityAndComplianceV2/README.md

```diff
@@ -1,113 +1,193 @@
+  - **App ID and Tenant ID**: Ensure these are correctly copied from your Azure app registration.
+  - **App Secret**: Verify that the secret has been correctly entered and has not expired.
+  - **UPN/Email**: Ensure that the email address is correctly formatted and belongs to a user with the necessary permissions.
+
+**Additional Steps:**
+- **Test with a Different Account**: If possible, try using a different user account to verify if the issue is related to specific user permissions.
+
+#### 3. Failed OAuth2.0 Authorization
+
+**Scenario:** After running the `!o365-sc-auth-start` command, the authorization process fails, and the integration cannot authenticate with the Security & Compliance Center.
+
+**Solution:**
+- **Double-Check App Permissions**: Make sure that the app registration in Azure has the necessary permissions, specifically `Exchange.ManageAsApp`.
+- **Grant Admin Consent**: Ensure that admin consent has been granted for the required permissions. Without this, the application cannot function correctly.
+- **Review Authentication Setup**: Ensure that "Allow public client flows" is enabled in the Azure app registration settings under **Authentication**.
+
+**Additional Steps:**
+- **Use the `!o365-sc-auth-test` Command**: Run this command to verify if the integration can successfully authenticate. If this test fails, revisit the app registration settings and verify all configurations.
+
+### Testing the Integration
+
+1. **Test Basic Connectivity**:
+   - Use the `!o365-sc-auth-test` command to confirm that the integration can successfully authenticate with the Security & Compliance Center.
+
+2. **Run a Simple Search**:
+   - Execute the `!o365-sc-search` command with basic parameters to ensure the integration can perform a search operation. This helps verify that the configuration is correct and the integration can communicate with the Security & Compliance Center.
 
-            | **Parameter** | **Description**                                          | **Required** |
-            | ------------- | -------------------------------------------------------- | ------------ |
-            | url           | Search and Compliance URL                                | True         |
-            | credentials   | Fill **only** Email (aka UPN), Password should be empty. | False        |
-            | insecure      | Trust any certificate \(not secure\)                     | False        |
+3. **Check Logs and Outputs**:
+   - After running test commands, review the logs and command outputs in the War Room for any errors or warnings. This can provide additional insights into what might be going wrong.
 
-   2. Open playground -  War-room:
+4. **Review Azure Logs**:
+   - If issues persist, review the logs in Azure to identify any access issues or authentication errors. This can help diagnose problems related to Azure AD configuration.
```

Comment:
```suggestion
   If issues persist, review the logs in Azure to identify any access issues or authentication errors. This can help diagnose problems related to Azure AD configuration.
```